lib/quantity/math.ex

```diff
@@ -222,4 +222,56 @@ defmodule Quantity.Math do
   def round(quantity, decimal_count) do
     Quantity.new(Decimal.round(quantity.value, decimal_count, :half_up), quantity.unit)
   end
+
+  defp reduce_unit(unit) do
```

Comment:
Suggested comment: 

```
# reduce a unit to a standard form
# apply standard transformations, e.g. a/1 = a, etc.
# multiple-layer divisions will be reduced to a single fraction with numerators and denominators
```

It's actually pretty hard. Can it be used?

Reply:
:+1: Added in 82c6f4e